protocol-designer/src/feature-flags/reducers.ts

```diff
@@ -31,7 +31,9 @@ const initialFlags: Flags = {
     process.env.OT_PD_ENABLE_HOT_KEYS_DISPLAY === '1' || true,
   OT_PD_ENABLE_REACT_SCAN: process.env.OT_PD_ENABLE_REACT_SCAN === '1' || false,
   OT_PD_ENABLE_LIQUID_CLASSES:
-    process.env.OT_PD_ENABLE_LIQUID_CLASSES === '1' || false,
+    process.env.OT_PD_ENABLE_REACT_SCAN === '1' || false,
```

Comment:
i guess needs to merge edge into this branch since you fixed this already, don't you?

```suggestion
    process.env.OT_PD_ENABLE_LIQUID_CLASSES === '1' || false,
```

Reply:
oops ya good catch.